src/main/java/org/sonarlint/intellij/ui/ruledescription/RuleParsingUtils.kt

```diff
@@ -99,6 +109,34 @@ class RuleParsingUtils {
             return createScrollPane(mainPanel)
         }
 
+        fun addTab(project: Project, parent: Disposable, tabDesc: RuleDescriptionTabDto,
```

Comment:
I find the location a bit surprising, `RuleParsingUtils` is not really related to populating a tab in my opinion. We already do this for other components, what about having a dedicated class like `RuleDescriptionTabPanel` that would contain this logic?